.github/workflows/CI-notebook-text.yml

```diff
@@ -0,0 +1,111 @@
+name: CI Text Notebooks
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  ci-notebook-text:
+    env:
+      node-version: 16.x
+    strategy:
+      matrix:
+        operatingSystem: [ubuntu-latest, windows-latest]
+        pythonVersion: [3.7, 3.8, 3.9, "3.10"]
+
+    runs-on: ${{ matrix.operatingSystem }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.pythonVersion }}
+
+      - name: Use Node.js ${{ env.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.node-version }}
+
+      - name: Install yarn
+        run: npm install yarn -g
+
+      - name: Install yarn dependencies
+        run: |
+          yarn config set network-timeout 300000
+          yarn install --frozen-lock-file
+
+      - name: Build Typescript
+        run: yarn buildall
+
+      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
+        name: Install pytorch on non-MacOS
+        shell: bash -l {0}
+        run: |
+          conda install --yes --quiet pytorch captum cpuonly -c pytorch
+
+      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
```

Comment:
Would the macos tests be desirable to add for vision as well?

Reply:
I think we had some test issues so we removed them? We should probably try to add them back in.